doc/man/8/ceph-disk.rst

```diff
@@ -176,6 +193,62 @@ Usage::
 
 Here, [PATH] is path to a block device or a directory.
 
+deactivate
+----------
+Deactivate the Ceph OSD. It stops osd daemo and try to mark this osd out
+(optional). To prevent OSD activate by ``udev``, remove the *ready*, *active*,
+*INIT-specific* files and add the *deactive* file. Last, unmount the currect
```

Comment:
The content of the OSD is left untouched but the *ready*, *active*, *INIT-specific* files are removed (so that it is not automatically re-activated by the ``udev`` rules) and the file *deactive* is created to remember the OSD is deactivated.